Dockerfile

```diff
@@ -10,11 +10,15 @@ RUN npm install
 RUN npm run dev
 
 # Second stage from compatible php version
-FROM php:7.3
+FROM php:7.4
 
 # Copy container directory from previous stage
 COPY --from=0 /container .
 
+# Copy php-extension installer from Docker Hub image and install required php extensions
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/bin
+RUN install-php-extensions gd zip
+
 # Install composer and get php dependencies
-COPY --from=composer /usr/bin/composer /usr/bin/composer
+COPY --from=composer /usr/bin/composer /usr/bin
```

Comment:
The **Troubleshooting** section on [this Docker Hub page](https://hub.docker.com/_/composer) explains this command.